frontend/src/pages/IndividualSighting/IndividualSighting.tsx

```diff
@@ -1,9 +1,115 @@
+                            <h6 className="mb-0">Submitted by</h6>
+                        </div>
+                        <div className="col-sm-9">
+                            <p className="text-muted mb-0">{sighting.username}</p>
+                        </div>
+                    </div>
+                    <hr />
+                    <div className="row">
+                        <div className="col-sm-3">
+                            <h6 className="mb-0">Species</h6>
+                        </div>
+                        <div className="col-sm-9">
+                            <p className="text-muted mb-0">{sighting.speciesName}</p>
+                        </div>
+                    </div>
+                    <hr />
+                    <div className="row">
+                        <div className="col-sm-3">
+                            <h6 className="mb-0">Longitude</h6>
+                        </div>
+                        <div className="col-sm-9">
+                            <p className="text-muted mb-0">{sighting.longitude}</p>
+                        </div>
+                    </div>
+                    <hr />
+                    <div className="row">
+                        <div className="col-sm-3">
+                            <h6 className="mb-0">Latitude</h6>
+                        </div>
+                        <div className="col-sm-9">
+                            <p className="text-muted mb-0">{sighting.latitude}</p>
+                        </div>
+                    </div>
+                    <hr />
+                    <div className="row">
+                        <div className="col-sm-3">
+                            <h6 className="mb-0">Description</h6>
+                        </div>
+                        <div className="col-sm-9">
+                            <p className="text-muted mb-0">{sighting.description}</p>
+                        </div>
+                    </div>
+                    <hr />
+                    <div className="row">
+                        <div className="col-sm-3">
+                            <h6 className="mb-0">Submitted at</h6>
+                        </div>
+                        <div className="col-sm-9">
+                            <p className="text-muted mb-0">{sighting.dateTime}</p>
+                        </div>
+                    </div>
+                </div>
+                <div className="row g-0">
+                    <div className="col-6 p-2">
+                        <button id="update-button" data-testid="update-button" className="btn btn-primary btn-md w-100">
+                            Update
+                        </button>
+                    </div>
+                    <div className="col-6 p-2">
+                        <button id="delete-button" data-testid="delete-button" className="btn btn-primary btn-md w-100">
+                            Delete
+                        </button>
```

Comment:
This button should only show if the sighting belongs to this user, please can you add some conditions so it only renders in those circumstances?